hz.ps1

```diff
@@ -494,6 +494,28 @@ function ensure-command($command) {
     }
 }
 
+function get-master-server-version ( $result ) {
```

Comment:
the `hz.ps1` script can be used to run the remote controller as such: `hz.ps1 run-remote-controller -server-version <version>` where `<version>` is the version of the server that you want to run. And then it will fetch that version from Maven. We wanted a way (when 5.1 was not ready yet) to say: run the version that is on the "master" branch of the `hazelcast` project. So today if I run `hz.ps1 run-remote-controller -server-version master` I'll get the latest 5.2 snapshot I believe. Again, that was because at one moment it was the only possible way to run the 5.1 server.

Reply:
let's talk. I still did not get it.